assets/js/Services/Contrast.js

```diff
@@ -225,6 +225,14 @@ class Contrast {
     return '#' + color
   }
 
+  convertHtmlRgb2Hex(html) {
+    return html.replace(/rgb\((.+?)\)/ig, (_, rgb) => {
+      return '#' + rgb.split(',')
+        .map(str => parseInt(str, 10).toString(16).padStart(2, '0'))
+        .join('')
+    })
+  }
```

Comment:
This will also replace all occurrences of rgb color values in the text of the page, which could be bad for any classes that are teaching CSS or HTML.  I suggest something like this, which would target only rgb color values that are within an HTML tag:
```js
convertHtmlRgb2Hex(html) {
  return html.replace(/(<[a-z"=;#:\-\d\s]*rgb\()(.+?)(\)+[a-z"=;#:\-\s]*>)/ig, (_, pre, rgb, post) => {
    return pre + '#' + rgb.split(',')
      .map(str => parseInt(str, 10).toString(16).padStart(2, '0'))
      .join('') + post
  })
}
```

You can try it out here:  https://codepen.io/bagofarms/pen/ExXoOXe

Reply:
That's a great suggestion. That specific implementation won't quite work when a tag has multiple attributes with an rgb value such as color and background-color, and the parentheses are kept around. 

I went ahead and fixed those issues and implemented the change to ignore rgb elements that aren't inside html tags.